contents/docs/user-guides/trends.mdx

```diff
@@ -1,58 +1,324 @@
+Some operations allow you to pass multiple values to compare against, in which case the event will be included if any of these values return true when compared.
+
+#### Types of filters
+
+Within trends there are two different ways of filtering down events:
+
+#### Inline filters
+
+![example inline filter](../../images/docs/user-guides/trends/inline-filter.png)
+
+These filters apply to a specific series within a graph, and different series can have different inline filters.
+
+> Currently, you can only combine inline filters with `AND` operations. If you want to include events that match _at least one_ of a group of filters, you can use [filter groups](#filter-groups)
+
+#### Filter groups
+
+Filter groups allow you to apply filters to **all series** within an insight. These filter groups are composed of a number of single filter, which are can be combined in the following two ways:
+
+-   `AND` - Events have to match _every_ filter within the group
+-   `OR` - Events only have to match _a single_ filter within the group
+
+![example global filter](../../images/docs/user-guides/trends/filter-groups.png)
+
+One thing to note about global filters: since they apply to all events within an insight, and these events may not share all their properties, global filters allow you to use any property that has been seen in _at least one_ of your events.
+If you happen to create a filter that references a property that not all events have, events that do not contain this property will automatically fail the check and will be excluded by the filter.
+
+#### Filtering internal and test users
+
+The final option for filtering down events is using the option to filter out 'internal and test users'.
+This option is useful for excluding events sent from local builds of your product, or by members of your team that you don't want to include in your analysis.
+
+![filtering internal and test users turned on](../../images/docs/user-guides/trends/internal-and-test-users.png)
+
+If you want to customize the filters that are used, you can click on the settings icon.
+
+## Breaking down by properties
+
+Breakdowns offer a way for you to split up your trends graph by the values of a certain property. This is a very powerful tool when analyzing a trend and trying to determine the cause of a change in one of your metrics.
+Once you've selected a property to break down on, you should see your graph split into multiple lines, each one corresponding to a distinct value of that property.
+
+![breaking down an insight by browser](../../images/docs/user-guides/trends/breakdown.png)
+
+For readability, PostHog will only load the first 25 values of a breakdown when you first load an insight. You can manually load more values by clicking the 'Load more breakdown values' button.
```

Comment:
Maybe something about visualisation here? that if you don't want to see multiple lines, but total counts, switch to a table view

Reply:
and a link to the visualisation options mentioned below

Reply:
Totally, I added a short section with a sample of a bar graph and linked to the table and value view options